server/http.go

```diff
@@ -309,22 +296,20 @@ func (p *Plugin) handleStartMeeting(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	_, appErr = p.API.GetChannelMember(req.ChannelID, userID)
-	if appErr != nil {
+	if _, appErr = p.API.GetChannelMember(req.ChannelID, userID); appErr != nil {
 		http.Error(w, "Forbidden", http.StatusForbidden)
 		return
 	}
 
-	if r.URL.Query().Get("force") == "" {
+	if forceCreate := r.URL.Query().Get("force"); forceCreate == "" {
```

Comment:
How does this make the code more readable then 
```suggestion
	if r.URL.Query().Get("force") == "" {
```
?

Reply:
This change was done in mickmister's PR https://github.com/mattermost/mattermost-plugin-zoom/pull/111.

Reply:
Actually, #111 didn't include that change: https://github.com/mattermost/mattermost-plugin-zoom/pull/111/files#diff-52d328bdc6be1ea07f80fdb67869a272R318